scripts/verify-nupkgs.ps1

```diff
@@ -16,7 +16,7 @@ function Verify-Nuget-Packages($packageDirectory, $version)
         "Microsoft.NET.Test.Sdk" = 16;
         "Microsoft.TestPlatform" = 603;
         "Microsoft.TestPlatform.Build" = 21;
-        "Microsoft.TestPlatform.CLI" = 477;
+        "Microsoft.TestPlatform.CLI" = 476;
```

Comment:
This file is no longer part of the nuget package: `Microsoft.TestPlatform.CLI.17.6.0-dev\contentFiles\any\netcoreapp3.1\Microsoft.TestPlatform.VsTestConsole.TranslationLayer.xml`